go/status/log_send.go

```diff
@@ -64,6 +66,37 @@ type LogSendContext struct {
 	processesLog     string
 }
 
+var noncharacterRxx = regexp.MustCompile(`[^\w]`)
+
+const redactedReplacer = "[REDACTED]"
+const serialPaperKeyWordThreshold = 5
+
+func redactPotentialPaperKeys(s string) string {
+	words := strings.Split(noncharacterRxx.ReplaceAllLiteralString(s, " "), " ")
+	didRedact := false
+	start := -1
+	for idx, word := range words {
+		if !libkb.ValidSecWord(word) {
+			start = -1
+			continue
+		}
+		if start == -1 {
+			start = idx
+		} else if idx-start+1 == serialPaperKeyWordThreshold {
+			for jdx := start; jdx <= idx; jdx++ {
+				words[jdx] = redactedReplacer
+			}
+			didRedact = true
+		} else if idx-start+1 > serialPaperKeyWordThreshold {
+			words[idx] = redactedReplacer
+		}
+	}
+	if didRedact {
```

Comment:
seems a little unfortunate that we lose all the punctuation here.

Reply:
I made a commit to address this. Don't know if it could be cleaner though.